packages/astro/src/vite-plugin-astro/hmr.ts

```diff
@@ -106,9 +146,8 @@ export async function handleHotUpdate(ctx: HmrContext, config: AstroConfig, logg
 			}
 		}
 	}
-	const isSelfAccepting = mods.every((m) => m.isSelfAccepting || m.url.endsWith('.svelte'));
 
-	const file = ctx.file.replace(config.root.pathname, '/');
+	const isSelfAccepting = mods.every((m) => m.isSelfAccepting || m.url.endsWith('.svelte'));
```

Comment:
I know this isn't new code, but can you add a comment to explain why the `.svelte` extension is important here?